cmd/server-main.go

```diff
@@ -277,7 +280,14 @@ func mergeServerCtxtFromConfigFile(configFile string, ctxt *serverCtxt) error {
 		ctxt.SFTP = append(ctxt.SFTP, fmt.Sprintf("ssh-private-key=%s", cf.Options.SFTP.SSHPrivateKey))
 	}
 
-	ctxt.Layout, err = buildDisksLayoutFromConfFile(cf.Pools)
+	if cf.EnableExpandPools && len(cf.Pools) != 1 {
+		return fmt.Errorf("only one pool is supported for expand pool")
```

Comment:
`pool` word used twice. better could be `only one pool is supported for expansion`